README.md

```diff
@@ -9,13 +9,45 @@ Mangata XCM Auto-compound E2E Demo
 3. Run a local Rococo-Mangata-Turing network with instructions in [Run local network with Zombienet](https://github.com/OAK-Foundation/OAK-blockchain#quickstart-run-local-network-with-zombienet)
 
 # Run Mangata demo
-1. Set up accounts and pools
+1. Launch OAK-blockchain, Rococo and Mangata.
+
+	- Compile OAK
+
+		https://github.com/OAK-Foundation/OAK-blockchain/
+
+		```
+		cargo build --release --features turing-node --features dev-queue
+		```
+
+
+	- Compile modified Mangata
+
+		https://github.com/OAK-Foundation/mangata-node/tree/mangata-demo
+
+		```
+		cargo build --release --features mangata-rococo
+		```
+
+
+	- Clear the mangata reply chain data
```

Comment:
Update to,
`Clear the relay chain directory of mangata-node before running zombienet command, because currently mangata-node couldn’t auto-purge.`